### ISSUE TITLE
Consider removing `TagHelperContent.GetContent()`

### ISSUE THREAD

**Issue by sornaks** [aspnet/Razor#479]
`TagHelperContent.GetContent()` is hardcoded to use the default encoder, which is bad because it's a replaceable service. We recently added `TagHelperContent.GetContent(IHtmlEncoder)` - we should remove the no-param version.

Original Text:
~~Currently the content which is stored in `TagHelperContent.GetContent()` is unencoded. The only way to get encoded content is using `WriteTo(TextWriter, IHtmlEncoder)`. Consider having an overload like `GetContent(IHtmlEncoder)` which returns the encoded content.~~

~~Reference PR comment: https://github.com/aspnet/Razor/pull/473/files#r36673441~~

**Comment by rynowak** [comment]
See discussion here: https://github.com/aspnet/Razor/pull/541/files#r40223087

**Comment by Eilon** [comment]
Close?

**Comment by NTaylorMullen** [comment]
@rynowak do we still feel that this is important? It's relatively simple to do.

**Comment by rynowak** [comment]
It's not really that important, especially since we have the overload that lets you pass in the encoder. If you get blocked on this it's easy to work around.

**Comment by NTaylorMullen** [comment]
kk, going to close as wontfix since it's an easy mechanism to use/workaround if it causes you trouble.
